src/app/components/nav/Nav.js

```diff
@@ -25,13 +26,11 @@ function NavigationBar() {
 						<NavLinkItem end to="/">
 							Home
 						</NavLinkItem>
-						<NavLinkItem to="/starter-packs">Resources</NavLinkItem>
-						<NavLinkItem to="/schedule">Schedule</NavLinkItem>
-						{/*<NavLinkItem to={{ pathname: APPLICATION_URL }}*/}
-						{/*             target="_blank"*/}
-						{/*             className="nav-link-outline">*/}
-						{/*    Apply*/}
-						{/*</NavLinkItem>*/}
+						{/* <NavLinkItem to="/starter-packs">Resources</NavLinkItem> */}
+						{/* <NavLinkItem to="/schedule">Schedule</NavLinkItem> */}
+						<NavLinkItem as="a" href={APPLICATION_URL} target="_blank" rel="noreferrer">
+							<ExternalLink text="Apply" />
+						</NavLinkItem>
```

Comment:
The current implementation of the `ExternalLink` component includes an anchor element of its own, so the current composition causes invalid DOM nesting (anchor should not be child of another anchor). We could look into manipulating the props of `NavLinkItem`, but it might be simpler to instead have an `ExternalLinkIcon` component which can be used independently.